src/applications/personalization/profile-2/components/Profile2Wrapper.jsx

```diff
@@ -83,4 +90,4 @@ Profile2.propTypes = {
   ).isRequired,
 };
 
-export default Profile2;
+export default connect(mapStateToProps)(Profile2Wrapper);
```

Comment:
even if we don't need it, might be nice to add an export of the unconnected version: `export {Profile2Wrapper}`